frank-flow/src/frontend/src/app/flow/options/options.component.ts

```diff
@@ -26,10 +27,12 @@ export class OptionsComponent implements OnInit, OnDestroy {
   selectedAttribute!: any;
   newAttributeValue!: string;
   nodeName!: string | undefined;
-  nodeDescription?: string;
+  element?: any;
```

Comment:
In other files you changed member variables to be private or public, I think we should stay consistent with that.